_pages/en/strategy-build-vs-buy.md

```diff
@@ -322,23 +425,27 @@ The following traceability matrix is used to show alignment with various strateg
 </tr>
 </table>
 
+## Appendix C - References
+
+[^1]: Pia Andrews, in [What improving IT procurement in the public sector looks like: navigating, balancing, and initiating procurement conversations to avoid sprinting off cliffs](https://www.themandarin.com.au/118548-improving-it-procurement-public-sector-navigating-balancing-initiating-procurement-conversations/), writes that many COTS implementations “require significant customization, configuration, integration, or modifications to actually implement a COTS product into production.”
+
 ## Appendix D - Definitions
 
 IT Solution
-: An IT Solution is the combination of 1 or many IT Products, which are in turn comprised of 1 or many Software and/or Hardware, obtained through many possible ways: built internally, obtained as open source, provided by a company as an executable application under a proprietary licence, as a standalone device, or used as a service through a subscription model. (See the following standard IITB definition.)
+: An IT Solution is the combination of 1 or many IT Products, which are in turn comprised of 1 or many Software and/or Hardware, acquired through many possible ways: built internally, acquired as open source, provided by a company as an executable application under a proprietary licence, as a standalone device, or used as a service through a subscription model. (See the following standard IITB definition.)
```

Comment:
It seems like we have removed numbers in other areas (3rd to third). So thinking this should read '....the combination of one or many IT Products'.  Also '...which are in turn comprised of one or many Software...'

Reply:
Yes, will update!